projects/plugins/jetpack/modules/masterbar/admin-menu/menu-mappings.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$menu_mappings = array(
```

Comment:
To fix this small linting issue you could return the array directly; PHP supports to return an array in the php script.

So, instead of ```$menu_mappings = array(``` you could have something like ```return array(```

Reply:
I'm also fine without this change. :)